changelog_unreleased/TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+<!--
+
+1. Choose a folder based on which language your PR is for.
+
+   - For JavaScript, choose `javascript/` etc.
+   - For TypeScript specific syntax, choose `typescript/`.
+   - If your PR applies to multiple languages, such as TypeScript/Flow, choose one folder and mention which languages it applies to.
+
+2. In your chosen folder, create a file with your PR number: `pr-XXXX.md`. For example: `typescript/pr-6728.md`.
+
+3. Copy the content below and paste it in your new file.
+
+4. Fill in a title, the PR number and your user name.
+
+5. Optionally write a description. Many times it’s enough with just sample code.
+
+6. Change ```jsx to your language. For example, ```yaml.
+
+7. Change the `// Input` and `// Output` comments to the comment syntax of your language. For example, `# Input`.
+
+8. Choose some nice input example code. Paste it along with the output before and after your PR.
+
+-->
+
+#### Title ([#XXXX](https://github.com/prettier/prettier/pull/XXXX) by [@user](https://github.com/user))
```

Comment:
It seems that the parentheses are unnecessary when attaching the PR and user link. The parentheses do not exist in the [current `CHANGELOG.unreleased.md`](https://raw.githubusercontent.com/prettier/prettier/master/CHANGELOG.unreleased.md) like below:

```md
#### Title ([#XXXX] by [@user])
```

Reply:
I don’t understand what you mean?

Reply:
Sorry...
```suggestion
#### Title ([#XXXX] by [@user])
```

Reply:
Then we have to add this at the bottom:

```md
[#XXXX]: https://github.com/prettier/prettier/pull/XXXX
[@user]: https://github.com/user
```

Which means more typing for contributors and more room for error.

Reply:
I see. I didn't know, sorry.